OpenRA.Mods.Common/Traits/World/EditorActorPreview.cs

```diff
@@ -70,7 +70,9 @@ public EditorActorPreview(WorldRenderer worldRenderer, string id, ActorReference
 				Footprint = new ReadOnlyDictionary<CPos, SubCell>(footprint);
 			}
 
-			var tooltip = Info.TraitInfoOrDefault<EditorOnlyTooltipInfo>() as TooltipInfoBase ?? Info.TraitInfoOrDefault<TooltipInfo>();
+			var tooltip = Info.TraitInfos<EditorOnlyTooltipInfo>().FirstOrDefault() as TooltipInfoBase
+				?? Info.TraitInfos<TooltipInfo>().FirstOrDefault();
```

Comment:
Note to self: This should probably be `.First(Exts.IsTraitEnabled)` too, in any case it should match `ActorSelectorLogic`.